pkg/scanners/trivy/types.go

```diff
@@ -14,6 +14,10 @@ import (
 	"github.com/eraser-dev/eraser/pkg/utils"
 )
 
+// currentExecutingLookPath is a variable that points to exec.LookPath by default,
+// but can be overridden for testing purposes.
+var currentExecutingLookPath = exec.LookPath
```

Comment:
It's a little weird to have this as a global variable here. Why not just use `exec.LookPath` in the `findTrivyExecutable` function?

Instead of overriding the lookup function for testing, why not create a `t.TempDir()`, create some executable files in it, and maniuplate `PATH` by using `os.Setenv`?